### PR TITLE
new: enable btf dynamic support.

### PR DESCRIPTION
**Pull request by FedeDP**
Also, cleanup ansible output in cleanup role.

### CODE REVIEW

File: roles/scap-open-test/tasks/main.yml
Comment (FedeDP) [inline roles/scap-open-test/tasks/main.yml:1]: Also, while doing this, i noticed that we have 2 task unloading the kmod; one is executed right before building the kmod.
Is this wanted or a leftover? @alacuku

**Comment by FedeDP** [comment]
Workflow run from this branch: https://github.com/alacuku/e2e-falco-tests/actions/runs/5586509736

**Comment by FedeDP** [comment]
Output:
# amazonlinux2-4.19
```
cat scap-open-and-modern-probe.json 
{
    "changed": false,
    "false_condition": "btf_support.stat.exists",
    "skip_reason": "Conditional result was False",
    "skipped": true
}
```
and
```
cat cmake-build-directory.json  | grep cmd
    "cmd": "cmake -DUSE_BUNDLED_DEPS=ON -DBUILD_LIBSCAP_MODERN_BPF=False -DBUILD_LIBSCAP_GVISOR=OFF -DBUILD_BPF=true -DCREATE_TEST_TARGETS=OFF ..\n",
```

# archlinux-6.0

```
cat scap-open-and-modern-probe.json 
{
    "changed": true,
    "cmd": "./libscap/examples/01-open/scap-open --num_events 50 --modern_bpf\n",
    "delta": "0:00:00.269259",
    "end": "2023-07-18 06:01:18.504678",
    "failed": false,
    "msg": "",
    "rc": 0,
    "start": "2023-07-18 06:01:18.235419",
...
```
and
```
cat cmake-build-directory.json  | grep cmd
    "cmd": "cmake -DUSE_BUNDLED_DEPS=ON -DBUILD_LIBSCAP_MODERN_BPF=ON -DBUILD_LIBSCAP_GVISOR=OFF -DBUILD_BPF=true -DCREATE_TEST_TARGETS=OFF ..\n",

```

**Comment by FedeDP** [comment]
Please note that there is something wrong with the nodes right now;
```
failed: [localhost] (item={'name': 'ubuntu-6.3', 'kernel': 'therealbobo/ubuntu-kernel:6.3-x86_64', 'rootfs': 'therealbobo/ubuntu-image:6.3-x86_64', 'arch': 'x86_64'}) => {"ansible_loop_var": "item", "changed": true, "cmd": "ignite run --config \"./roles/bootstrap/files/ubuntu-6.3.yaml\" --runtime docker\n", "delta": "0:00:00.020951", "end": "2023-07-18 12:33:52.555352", "item": {"arch": "x86_64", "kernel": "therealbobo/ubuntu-kernel:6.3-x86_64", "name": "ubuntu-6.3", "rootfs": "therealbobo/ubuntu-image:6.3-x86_64"}, "msg": "non-zero return code", "rc": 1, "start": "2023-07-18 12:33:52.534401", "stderr": "", "stderr_lines": [], "stdout": "time=\"2023-07-18T12:33:52Z\" level=fatal msg=\"v1alpha4.VM.ObjectMeta: runtime.ObjectMeta.Labels: ReadString: expects \\\" or n, but found 5, error found in #10 byte of ...|\\\":{\\\"run\\\":5587452723}|..., bigger context ...|1alpha4\\\",\\\"kind\\\":\\\"VM\\\",\\\"metadata\\\":{\\\"labels\\\":{\\\"run\\\":5587452723},\\\"name\\\":\\\"ubuntu-6.3-5587452723\\\"},\\\"spec\\\":|...\"", "stdout_lines": ["time=\"2023-07-18T12:33:52Z\" level=fatal msg=\"v1alpha4.VM.ObjectMeta: runtime.ObjectMeta.Labels: ReadString: expects \\\" or n, but found 5, error found in #10 byte of ...|\\\":{\\\"run\\\":5587452723}|..., bigger context ...|1alpha4\\\",\\\"kind\\\":\\\"VM\\\",\\\"metadata\\\":{\\\"labels\\\":{\\\"run\\\":5587452723},\\\"name\\\":\\\"ubuntu-6.3-5587452723\\\"},\\\"spec\\\":|...\""]}
```

File: roles/scap-open-test/tasks/main.yml
Comment (alacuku) [reply roles/scap-open-test/tasks/main.yml:1]: It's just to make sure that the module has been unloaded in case the plabooks is manually played.

**Pull request merged**